xml/deployment_prep_power.xml

```diff
@@ -25,16 +12,523 @@ fate#304108: PPC64 Hotplug memory remove
+      </para>
+    </step>
+    <step>
+      <para>
+	Next, choose the desired product registration method and complete registration.
+	If you register the system using the &scc;, you are prompted to enable
+	update repositories. Press <literal>Yes</literal>.
+      </para>
+      <informalfigure>
+      <mediaobject>
+       <imageobject>
+        <imagedata width="80%" fileref="install_sle_power_reg.png"/>
+       </imageobject>
+       <textobject><phrase>Registering the product</phrase>
+       </textobject>
+      </mediaobject>
+     </informalfigure>
+    </step>
+    <step>
+      <para>
+	Select the extensions and modules you wish to install. To select and deselect
+	a module or extensions, select it  using the arrow keys and press <keycap function="space"/>.
+	Depending on what extensions and  modules you select, you may be prompted to import
+	GnuPG keys for the associated repositories. 
+      </para>
+      <informalfigure>
+      <mediaobject>
+       <imageobject>
+        <imagedata width="80%" fileref="install_sle_power_ext.png"/>
+       </imageobject>
+       <textobject><phrase>Selecting extensions and modules</phrase>
+       </textobject>
+      </mediaobject>
+     </informalfigure>
+    </step>
+    <step>
+      <para>
+	Install the desired add-on products. If you choose to install an add-on,
+	you need to specify the installation source for it.
+      </para>
+      <informalfigure>
+      <mediaobject>
+       <imageobject>
+        <imagedata width="80%" fileref="install_sle_power_addon.png"/>
+       </imageobject>
+       <textobject><phrase>Installing add-on products</phrase>
+       </textobject>
+      </mediaobject>
+     </informalfigure>
+    </step>
+    <step>
+      <para>
+	Specify a partition scheme for your installation. To accept the default proposal,
+	press <literal>Next</literal> or press <keycombo><keycap function="alt"/><keycap>N</keycap></keycombo>.
+      </para>
+      <informalfigure>
+      <mediaobject>
+       <imageobject>
+        <imagedata width="80%" fileref="install_sle_power_part.png"/>
+       </imageobject>
+       <textobject><phrase>Specifying a partition scheme</phrase>
+       </textobject>
+      </mediaobject>
+     </informalfigure>
+    </step>
+    <step>
+      <para>
+	Choose the system role suitable for your particular scenario.
+      </para>
+      <informalfigure>
+      <mediaobject>
+       <imageobject>
+        <imagedata width="80%" fileref="install_sle_power_sysrole.png"/>
+       </imageobject>
+       <textobject><phrase>Choosing a system role</phrase>
+       </textobject>
+      </mediaobject>
+     </informalfigure>
+    </step>
+    <step>
+      <para>
+	The next several screen allow you to specify the appropriate
+	time zone, and create a user. If you choose not to create a user,
+	you are prompted to specify a root password.
+      </para>
+    </step>
+    <step>
+      <para>
+	In the installation summary screen, make sure to enable the SSH service
```

Comment:
Either "make sure the SSH service is enabled" **or** "enable the SSH service" but not both.